docs/static/arcsight-module.asciidoc

```diff
@@ -0,0 +1,405 @@
+[role="xpack"]
+[[arcsight-module]]
+=== Logstash ArcSight Module
+
+++++
+<titleabbrev>ArcSight Module</titleabbrev>
+++++
+
+NOTE: The Logstash ArcSight module is an
+https://www.elastic.co/products/x-pack[X-Pack] feature under the Basic License
+and is therefore free to use. Please contact
+mailto:arcsight@elastic.co[arcsight@elastic.co] for questions or more
+information.
+
+The Logstash ArcSight module enables you to easily integrate with and augment
+the ArcSight Data Platform (ADP) to explore your data in Elasticsearch using
+Kibana visualizations. With a single command, the module taps directly into the
+ADP data stream, parses and indexes the security events into Elasticsearch, and
+installs a suite of Kibana dashboards to get you exploring your data
+immediately. 
+
+[role="screenshot"]
+image::static/images/arcsight-architecture-diagram-2017.png[ArcSight architecture]
+
+[[arcsight-architecture]]
+==== Deployment Architecture
+
+The Logstash ArcSight module understands CEF (Common Event Format), and can
+accept, enrich, and index these events for analysis on the Elastic Stack. ADP
+contains two core data collection components for data streaming:
+
+* The _EventBroker (EB)_ is the central hub for incoming data and is based on
```

Comment:
I should have changed this earlier, but lets add spaces here. "Event Broker" and "Smart Connectors"